src/components/addItems/ItemModalBackground.tsx

```diff
@@ -166,44 +211,32 @@ export default function ItemModalBackground({
             <div
               style={{
                 display: "flex",
+                flexWrap: "wrap",
                 flexDirection: "row",
-                paddingBottom: 30,
+                paddingBottom: 10,
                 justifyContent: "center",
               }}
             >
-              {/* Category Selectable Buttons */}
-              <div
-                style={{
-                  display: "flex",
-                  flexDirection: "row",
-                  paddingBottom: 30,
-                  justifyContent: "center",
-                }}
-              >
-                {/* Map tag names to their buttons */}
-                {[
-                  "Clothing",
-                  "Collectibles",
-                  "Jewlery",
-                  "Art",
-                  "Household Items",
-                ].map((tag) => (
-                  <button
-                    style={{
-                      marginRight: 10,
-                      width: tag === "Household Items" ? 150 : 100,
-                    }}
-                    type="button"
-                    className={
-                      selectedTags.includes(tag) ? "tag" : "tagSelected"
-                    }
-                    onClick={() => handleTagClick(tag)}
-                    key={tag}
-                  >
-                    {tag}
-                  </button>
-                ))}
-              </div>
+              {/* Map tags to categories name fields */}
+              {categories.map((category) => (
+                <button
+                  style={{
+                    marginRight: 10,
+                    marginBottom: 10,
+                    width: category.name === "Household Items" ? 150 : 100,
+                  }}
+                  type="button"
+                  className={
+                    selectedTags.includes(category.name!)
+                      ? "tag"
```

Comment:
Please rename this class to something else. It is interfering with the other tags in the table. Rename it in the css file as well.